package com.parksmart.repository;
import java.util.Optional; import org.springframework.data.jpa.repository.JpaRepository; import com.parksmart.entity.User;
public interface UserRepository extends JpaRepository<User,Long>{Optional<User> findByEmailIgnoreCase(String email);}